app/src/main/java/com/maq/xprize/onecourse/hindi/receivers/NotificationReminderReceiver.java

```diff
@@ -0,0 +1,81 @@
+package com.maq.xprize.onecourse.hindi.receivers;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+
+import com.maq.xprize.onecourse.hindi.R;
+import com.maq.xprize.onecourse.hindi.mainui.MainActivity;
+
+public class NotificationReminderReceiver extends BroadcastReceiver {
+    private static final String CHANNEL_ID = "studyReminderChannel";
+    private static final String LOGTAG = "ReminderNotification";
+    private static final int NOTIFICATION_ID = LOGTAG.hashCode();
+    private static final CharSequence notificationContentText = "It's time to study today's content. Let's start...";
```

Comment:
Can we get this as well from the strings.xml file?

Reply:
Yes, we can do that

Reply:
Used strings.xml